Frontend/app/routes/home.tsx

```diff
@@ -6,5 +14,19 @@ export function meta() {
 }
 
 export default function Home() {
-  return <p>hoi</p>;
+  /**
+   * TODO: Swap this out with tanstack query
```

Comment:
Gaan we tanstack gebruiken?

Reply:
Tanstack query is echt fenomenaal voor dit soort projecten